### ISSUE TITLE
Is there any way to fine-tune an existing SSD checkpoints on a cpu?

### ISSUE THREAD

**Issue by bousejin** [balancap/SSD-Tensorflow#75]
I just changed data_format to 'NHWC' on train_ssd_network file and used this commands:

DATASET_DIR=./tfrecords/
TRAIN_DIR=./logs/ssd_512_voc07/
CHECKPOINT_PATH=./checkpoints/VGG_VOC0712_SSD_512x512_ft_iter_120000.ckpt

python train_ssd_network.py 
    --train_dir=${TRAIN_DIR} 
    --dataset_dir=${DATASET_DIR} 
    --dataset_name=pascalvoc_2007 
    --dataset_split_name=train 
    --model_name=ssd_512_vgg 
    --checkpoint_path=${CHECKPOINT_PATH} 
    --save_summeries_secs=60 
    --save_internal_secs=600 
    --weight_decay=0.0005 
    --optimizer=adam 
    --learning_rate=0.001 
    --batch_size=32

And the output is:

# =========================================================================== #
# Training | Evaluation flags:
# =========================================================================== #
{'adadelta_rho': 0.95,
 'adagrad_initial_accumulator_value': 0.1,
 'adam_beta1': 0.9,
 'adam_beta2': 0.999,
 'batch_size': 32,
 'checkpoint_exclude_scopes': None,
 'checkpoint_model_scope': None,
 'checkpoint_path': './checkpoints/VGG_VOC0712_SSD_512x512_ft_iter_120000.ckpt',
 'clone_on_cpu': False,
 'dataset_dir': './tfrecords/',
 'dataset_name': 'pascalvoc_2007',
 'dataset_split_name': 'train',
 'end_learning_rate': 0.0001,
 'ftrl_initial_accumulator_value': 0.1,
 'ftrl_l1': 0.0,
 'ftrl_l2': 0.0,
 'ftrl_learning_rate_power': -0.5,
 'gpu_memory_fraction': 0.8,
 'ignore_missing_vars': False,
 'label_smoothing': 0.0,
 'labels_offset': 0,
 'learning_rate': 0.001,
 'learning_rate_decay_factor': 0.94,
 'learning_rate_decay_type': 'exponential',
 'log_every_n_steps': 10,
 'loss_alpha': 1.0,
 'match_threshold': 0.5,
 'max_number_of_steps': None,
 'model_name': 'ssd_512_vgg',
 'momentum': 0.9,
 'moving_average_decay': None,
 'negative_ratio': 3.0,
 'num_classes': 21,
 'num_clones': 1,
 'num_epochs_per_decay': 2.0,
 'num_preprocessing_threads': 4,
 'num_readers': 4,
 'opt_epsilon': 1.0,
 'optimizer': 'adam',
 'preprocessing_name': None,
 'rmsprop_decay': 0.9,
 'rmsprop_momentum': 0.9,
 'save_interval_secs': 600,
 'save_summaries_secs': 600,
 'train_dir': './logs/ssd_512_voc07/',
 'train_image_size': None,
 'trainable_scopes': None,
 'weight_decay': 0.0005}

# =========================================================================== #
# SSD net parameters:
# =========================================================================== #
{'anchor_offset': 0.5,
 'anchor_ratios': [[2, 0.5],
                   [2, 0.5, 3, 0.3333333333333333],
                   [2, 0.5, 3, 0.3333333333333333],
                   [2, 0.5, 3, 0.3333333333333333],
                   [2, 0.5, 3, 0.3333333333333333],
                   [2, 0.5],
                   [2, 0.5]],
 'anchor_size_bounds': [0.1, 0.9],
 'anchor_sizes': [(20.48, 51.2),
                  (51.2, 133.12),
                  (133.12, 215.04),
                  (215.04, 296.96),
                  (296.96, 378.88),
                  (378.88, 460.8),
                  (460.8, 542.72)],
 'anchor_steps': [8, 16, 32, 64, 128, 256, 512],
 'feat_layers': ['block4',
                 'block7',
                 'block8',
                 'block9',
                 'block10',
                 'block11',
                 'block12'],
 'feat_shapes': [(64, 64), (32, 32), (16, 16), (8, 8), (4, 4), (2, 2), (1, 1)],
 'img_shape': (512, 512),
 'no_annotation_label': 21,
 'normalizations': [20, -1, -1, -1, -1, -1, -1],
 'num_classes': 21,
 'prior_scaling': [0.1, 0.1, 0.2, 0.2]}

# =========================================================================== #
# Training | Evaluation dataset files:
# =========================================================================== #
['./tfrecords/voc_2007_train_000.tfrecord',
 './tfrecords/voc_2007_train_001.tfrecord',
 './tfrecords/voc_2007_train_002.tfrecord',
 './tfrecords/voc_2007_train_003.tfrecord',
 './tfrecords/voc_2007_train_004.tfrecord',
 './tfrecords/voc_2007_train_005.tfrecord',
 './tfrecords/voc_2007_train_006.tfrecord',
 './tfrecords/voc_2007_train_007.tfrecord',
 './tfrecords/voc_2007_train_008.tfrecord',
 './tfrecords/voc_2007_train_009.tfrecord',
 './tfrecords/voc_2007_train_010.tfrecord',
 './tfrecords/voc_2007_train_011.tfrecord',
 './tfrecords/voc_2007_train_012.tfrecord',
 './tfrecords/voc_2007_train_013.tfrecord',
 './tfrecords/voc_2007_train_014.tfrecord',
 './tfrecords/voc_2007_train_015.tfrecord',
 './tfrecords/voc_2007_train_016.tfrecord',
 './tfrecords/voc_2007_train_017.tfrecord',
 './tfrecords/voc_2007_train_018.tfrecord',
 './tfrecords/voc_2007_train_019.tfrecord',
 './tfrecords/voc_2007_train_020.tfrecord',
 './tfrecords/voc_2007_train_021.tfrecord',
 './tfrecords/voc_2007_train_022.tfrecord',
 './tfrecords/voc_2007_train_023.tfrecord',
 './tfrecords/voc_2007_train_024.tfrecord',
 './tfrecords/voc_2007_train_025.tfrecord',
 './tfrecords/voc_2007_train_026.tfrecord',
 './tfrecords/voc_2007_train_027.tfrecord',
 './tfrecords/voc_2007_train_028.tfrecord',
 './tfrecords/voc_2007_train_029.tfrecord',
 './tfrecords/voc_2007_train_030.tfrecord',
 './tfrecords/voc_2007_train_031.tfrecord',
 './tfrecords/voc_2007_train_032.tfrecord',
 './tfrecords/voc_2007_train_033.tfrecord',
 './tfrecords/voc_2007_train_034.tfrecord',
 './tfrecords/voc_2007_train_035.tfrecord',
 './tfrecords/voc_2007_train_036.tfrecord',
 './tfrecords/voc_2007_train_037.tfrecord',
 './tfrecords/voc_2007_train_038.tfrecord',
 './tfrecords/voc_2007_train_039.tfrecord',
 './tfrecords/voc_2007_train_040.tfrecord',
 './tfrecords/voc_2007_train_041.tfrecord',
 './tfrecords/voc_2007_train_042.tfrecord',
 './tfrecords/voc_2007_train_043.tfrecord',
 './tfrecords/voc_2007_train_044.tfrecord',
 './tfrecords/voc_2007_train_045.tfrecord',
 './tfrecords/voc_2007_train_046.tfrecord',
 './tfrecords/voc_2007_train_047.tfrecord',
 './tfrecords/voc_2007_train_048.tfrecord',
 './tfrecords/voc_2007_train_049.tfrecord']

INFO:tensorflow:Fine-tuning from ./checkpoints/VGG_VOC0712_SSD_512x512_ft_iter_120000.ckpt. Ignoring missing vars: False
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE3 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.1 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
INFO:tensorflow:Starting Session.
INFO:tensorflow:Starting Queues.
INFO:tensorflow:global_step/sec: 0
terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_alloc
Aborted (core dumped)
